components/venueMap.tsx

```diff
@@ -0,0 +1,185 @@
+import { useEffect, useRef, useState } from 'react'
+import { MapContainer, TileLayer, LayersControl, useMapEvents, ImageOverlay, GeoJSON } from 'react-leaflet'
+import 'leaflet/dist/leaflet.css'
+import L from 'leaflet'
+
+import { sponsorFeatures } from './venueMapData'
+
+import {} from 'leaflet-easybutton'
```

Comment:
minor: If this is imported for side-effects, you can do `import 'leaflet-easybutton'`